calico-enterprise_versioned_docs/version-3.18-2/variables.js

```diff
@@ -15,7 +15,7 @@ const variables = {
   noderunning: 'calico-node',
   rootDirWindows: 'C:\\TigeraCalico',
   registry: 'quay.io/',
-  chart_version_name: 'v3.18.0-2.0-0',
+  chart_version_name: 'v3.18.1',
```

Comment:
This doesn't seem right. I'd expect something like `v3.18.1-0`. Check with rene?